diff --git a/node_modules/react-devtools-core/dist/backend.js b/node_modules/react-devtools-core/dist/backend.js
index 2f31f5a..1cd2029 100644
--- a/node_modules/react-devtools-core/dist/backend.js
+++ b/node_modules/react-devtools-core/dist/backend.js
@@ -14340,10 +14340,6 @@ function hook_arrayLikeToArray(arr, len) { if (len == null || len > arr.length)
  *
  */
 function installHook(target) {
-  if (target.hasOwnProperty('__REACT_DEVTOOLS_GLOBAL_HOOK__')) {
-    return null;
-  }
-
   var targetConsole = console;
   var targetConsoleMethods = {};
 
